src/routes/authentication/Authentication.component.tsx

```diff
@@ -115,36 +183,30 @@ const Authentication = () => {
             open: true,
             type: "success",
           });
```

Comment:
We immediately redirect to welcome after signing in so don't see this snackbar. So I guess we should probably take it out